src/tile.cpp

```diff
@@ -20,8 +20,7 @@
 extern Game g_game;
 extern MoveEvents* g_moveEvents;
 
-StaticTile real_nullptr_tile(0xFFFF, 0xFFFF, 0xFF);
-Tile& Tile::nullptr_tile = real_nullptr_tile;
+std::shared_ptr<Tile> Tile::nullptr_tile = std::make_shared<StaticTile>(0xFFFF, 0xFFFF, 0xFF);
```

Comment:
Could this be `const auto`
```suggestion
const auto Tile::nullptr_tile = std::make_shared<StaticTile>(0xFFFF, 0xFFFF, 0xFF);
```